src/components/Organisms/MarketingPreferences/MarketingPreferences.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+
+import Text from '../../Atoms/Text/Text';
+import Link from '../../Atoms/Link/Link';
+import InputController from './InputController';
+import CheckAnswer from './CheckAnswer';
+import { Head, Body } from './MarketingPreferences.style';
+
+const MarketingPreferences = () => {
+  const { control, errors, watch, register } = useFormContext();
+
+  const watchEmail = watch('email', 'no');
+  const watchAddress = watch('address', 'no');
+  const watchPhone = watch('telephone', 'no');
+  const watchMobile = watch('mobilePhone', 'no');
+
+  return (
+    <>
+      <div>
+        <Head>
+          <Text tag="h3" size="l" family="Anton" uppercase weight="400">
+            Email
+          </Text>
+          <CheckAnswer name="email" register={register} />
+        </Head>
+        <Body>
+          <Text tag="p" color="grey">
+            You will receive the latest fundraising activities updates and
+            marketing material from Comic Relief brand.
```

Comment:
Is this the actual text or just placeholder? Wording is a little off I think

Reply:
I copied it from design. I will make it as child props so it can be editable.